man/cc-test-reporter-sum-coverage.1.md

```diff
@@ -21,6 +21,10 @@ Combine (sum) multiple pre-formatted coverage payloads into one.
 Output to *PATH*. If *-* is given, content will be written to *stdout*. Defaults
 to *coverage/codeclimate.json*.
 
+## -p, --parts *NUMBER*
+
+*NUMBER* of partial test coverage results. Required flag.
```

Comment:
How about

> Expect `$NUMBER` payloads to sum. If this many arguments are not present, `sum-coverage` will error. This ensures you don't accidentally sum incomplete results.

We shouldn't declare when things are optional or required in the individual options description. That should be conveyed via the top-level `SYNOPSIS`:

```
cc-test-reporter-sum-coverage --parts=<number> [--output=<path>] FILE [FILE, ...]
```